### PR TITLE
[IMP] l10n_ar: update document type names

### PR DESCRIPTION
**Pull request by bruno-zanotti**
Update the document type list names to match with the AFIP documentation
Task 302

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ar/data/l10n_latam.document.type.csv [hunk 1]
```diff
@@ -71,18 +71,18 @@ dc_ac_inc_cf,690,94,AJUSTES CONTABLES QUE INCREMENTAN EL CREDITO FISCAL,,,,,base
 dc_ac_dis_cf,700,95,AJUSTES CONTABLES QUE DISMINUYEN EL CREDITO FISCAL,,,,,base.ar,,False
 dc_f1116b,710,96,FORMULARIO 1116 B,,,,,base.ar,,False
 dc_f1116c,720,97,FORMULARIO 1116 C,,,,,base.ar,,False
-dc_oc_ncrg3419,730,99,OTROS COMP  QUE NO CUMPLEN CON LA R G  3419 Y SUS MODIF,,,invoice,OC-X,base.ar,not_zero,True
+dc_oc_ncrg3419,730,99,OTROS COMPROBANTES QUE NO CUMPLEN O ESTAN EXCEPTUADOS DE LA R.G. 1415 Y SUS MODIF ,,,invoice,OC-X,base.ar,not_zero,True
```
Comment (jjscarafia) [inline addons/l10n_ar/data/l10n_latam.document.type.csv:74]: @Bruno-Zanotti y would remove the spaces on `MODIF ` I know that afip table has this space, but better to remove it

File: addons/l10n_ar/data/l10n_latam.document.type.csv [hunk 2]
```diff
@@ -53,16 +53,16 @@ dc_cfcp,510,71,CREDITO FISCAL POR CONTRIBUCIONES PATRONALES,,,,,base.ar,,False
 dc_f1116,520,73,FORMULARIO 1116 RT,,,,,base.ar,,False
 dc_cptag,530,74,CARTA DE PORTE PARA EL TRANSPORTE AUTOMOTOR PARA GRANOS,,,,,base.ar,,False
 dc_cptfg,540,75,CARTA DE PORTE PARA EL TRANSPORTE FERROVIARIO PARA GRANOS,,,,,base.ar,,False
-dc_zeta,550,80,COMPROBANTE DIARIO DE CIERRE (ZETA),,,,,base.ar,,False
-dc_a_t,560,81,TIQUE FACTURA A   CONTROLADORES FISCALES,A,TIQUE FACTURA,invoice,TF-A,base.ar,not_zero,True
+dc_zeta,550,80,INFORME DIARIO DE CIERRE (ZETA) - CONTROLADORES FISCALES,,,,,base.ar,,False
+dc_a_t,560,81,TIQUE FACTURA A   ,A,TIQUE FACTURA,invoice,TF-A,base.ar,not_zero,True
```
Comment (jjscarafia) [inline addons/l10n_ar/data/l10n_latam.document.type.csv:57]: @Bruno-Zanotti y would remove the spaces on `A   ` I know that afip table has this space, but better to remove it

File: addons/l10n_ar/data/l10n_latam.document.type.csv (hunk 1 above)
Comment (Bruno-Zanotti) [reply addons/l10n_ar/data/l10n_latam.document.type.csv]: :+1:

File: addons/l10n_ar/data/l10n_latam.document.type.csv (hunk 2 above)
Comment (Bruno-Zanotti) [reply addons/l10n_ar/data/l10n_latam.document.type.csv]: :+1:

File: addons/l10n_ar/data/l10n_latam.document.type.csv [hunk 3]
```diff
@@ -22,22 +22,22 @@ dc_e_fs,200,22,FACTURAS - PERMISO EXPORTACION SIMPLIFICADO - DTO. 855/97,E,,,,ba
...
-dc_c_rg1415,260,36,"COMPROBANTES C DEL Anexo I, Apartado A, INC.F), R.G. N° 1415",C,,invoice,,base.ar,zero,False
-dc_nd_rg1415,270,37,NOTAS DE DEBITO O DOCUMENTO EQUIVALENTE QUE CUMPLAN CON LA R.G. N° 1415,,,debit_note,,base.ar,not_zero,False
-dc_nc_rg1415,280,38,NOTAS DE CREDITO O DOCMENTO EQUIVALENTE QUE CUMPLAN CON LA R.G. N° 1415,,,credit_note,,base.ar,not_zero,False
-dc_a_o_rg1415,290,39,OTROS COMPROBANTES A QUE CUMPLEN CON LA R G  1415,A,,invoice,OC-A,base.ar,not_zero,True
-dc_b_o_rg1415,300,40,OTROS COMPROBANTES B QUE CUMPLAN CON LA R.G. N° 1415,B,,invoice,OC-B,base.ar,zero,True
-dc_c_o_rg1415,310,41,OTROS COMPROBANTES C QUE CUMPLAN CON LA R.G. N° 1415,C,,invoice,OC-C,base.ar,zero,True
-dc_a_rf,320,50,RECIBO FACTURA A  REGIMEN DE FACTURA DE CREDITO,A,,,,base.ar,not_zero,False
+dc_a_rg1415,240,34,"COMPROBANTES A DEL APARTADO A, INC. F), R.G. Nº 1415",A,,invoice,,base.ar,not_zero,False
+dc_b_rg1415,250,35,"COMPROBANTES B DEL ANEXO I, APARTADO A, INC. F), R.G. Nº 1415",B,,invoice,,base.ar,zero,False
+dc_c_rg1415,260,36,"COMPROBANTES C DEL ANEXO I, APARTADO A, INC. F), R.G. Nº 1415",C,,invoice,,base.ar,zero,False
```
[the remark this answers is not in the document]
Comment (jjscarafia) [reply addons/l10n_ar/data/l10n_latam.document.type.csv:27]: @Bruno-Zanotti I would keep as in afip, but also no problem to change it.
But I would remove the double quotes "

"COMPROBANTES A DEL APARTADO A, INC. F), R.G. Nº 1415"
--> COMPROBANTES A DEL APARTADO A, INC. F), R.G. Nº 1415

File: addons/l10n_ar/data/l10n_latam.document.type.csv [hunk 4]
```diff
@@ -22,22 +22,22 @@ dc_e_fs,200,22,FACTURAS - PERMISO EXPORTACION SIMPLIFICADO - DTO. 855/97,E,,,,ba
...
+dc_c_o_rg1415,310,41,OTROS COMPROBANTES C QUE CUMPLAN CON LA R.G. Nº 1415,C,,invoice,OC-C,base.ar,zero,True
+dc_a_rf,320,50,RECIBO FACTURA A REGIMEN DE FACTURA DE CREDITO,A,,,,base.ar,not_zero,False
 dc_m_f,330,51,FACTURAS M,M,FACTURA,invoice,FA-M,base.ar,not_zero,True
 dc_m_nd,340,52,NOTAS DE DEBITO M,M,NOTA DE DEBITO,debit_note,ND-C,base.ar,not_zero,True
 dc_m_nc,350,53,NOTAS DE CREDITO M,M,NOTA DE CREDITO,credit_note,NC-C,base.ar,not_zero,True
 dc_m_r,360,54,RECIBOS M,M,RECIBO,invoice,RE-M,base.ar,not_zero,True
 dc_m_nvc,370,55,NOTAS DE VENTA AL CONTADO M,M,,,,base.ar,not_zero,False
-dc_m_rg1415,380,56,COMPROBANTES M DEL ANEXO I  APARTADO A  INC F   R G  N  1415,M,,invoice,,base.ar,not_zero,False
-dc_m_o_rg1415,390,57,OTROS COMPROBANTES M QUE CUMPLAN CON LA R G  N  1415,M,,invoice,,base.ar,not_zero,False
+dc_m_rg1415,380,56,"COMPROBANTES M DEL ANEXO I, APARTADO A, INC. F), R.G. Nº 1415",M,,invoice,,base.ar,not_zero,False
```
Comment (jjscarafia) [inline addons/l10n_ar/data/l10n_latam.document.type.csv:39]: same here, remove double quotes "

File: addons/l10n_ar/data/l10n_latam.document.type.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (Bruno-Zanotti) [reply addons/l10n_ar/data/l10n_latam.document.type.csv:27]: @jjscarafia ok, the purpose of the double quotes is to keep the commas as in AFIP, but are not shown in Odoo. Anyway we can remove both, result:

> COMPROBANTES A DEL APARTADO A INC. F) R.G. Nº 1415

What do yo think about it?

[the remark this answers is not in the document]
Comment (jjscarafia) [reply addons/l10n_ar/data/l10n_latam.document.type.csv:27]: ah, ok, sorry. It's ok then

**Comment by ren-odoo** [comment]
Hi @Bruno-Zanotti 

I tested it and it looks good, @jco-odoo this is ready for your review. 

Regards.

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**